melos.yaml

```diff
@@ -3,10 +3,6 @@ name: envied
 packages:
   - "**"
 
-command:
-  bootstrap:
-    usePubspecOverrides: true
```

Comment:
Why?

Reply:
I found this error in my IDE (check the melos VSCode extension):  
<img width="631" alt="image" src="https://github.com/user-attachments/assets/21e80bba-e078-449f-9dc7-36f3607cd414">

But thats bc it was deprecated in [version 3.0.0](https://pub.dev/packages/melos/changelog#300) in [this issue](https://github.com/invertase/melos/pull/430)